cmd/ddev/cmd/dev_ssh.go

```diff
@@ -32,8 +32,7 @@ var LocalDevSSHCmd = &cobra.Command{
 			"bash",
 		)
 		if err != nil {
-			log.Println(err)
-			util.Failed("Failed to run exec command.")
+			log.Fatal("Failed DockerCompose exec bash command", err)
```

Comment:
This should still use util.Failed() so we get scary red text, but I also changed this error in the unique sitename PR: https://github.com/drud/ddev/pull/169/files#diff-ddf6929ce4a24c0882819f3dc3e8dc2fR34 I'd prefer the message I wrote over this one, as I think it is clearer to the user that the ssh command failed. 

To me, "DockerCompose exec bash command" seems like info overload. It also assumes the user would understand what those concepts are. If any of that information is necessary to resolve the issue, I feel like it should be exposed in what `err` outputs in the most helpful way, if possible.